pkg/controller/scansettingbinding/scansettingbinding_controller.go

```diff
@@ -246,7 +281,21 @@ func (r *ReconcileScanSettingBinding) Reconcile(ctx context.Context, request rec
 	}
 
 	// The suite already exists, should we update?
-	if suiteNeedsUpdate(&suite, &found) {
+	if r.suiteNeedsUpdate(&suite, &found, instance) {
```

Comment:
Same as below, I feel we shouldn't need these because suite should already has latest scansetting from  https://github.com/ComplianceAsCode/compliance-operator/pull/396/files#diff-c647a52d3b0966655cdd2384a18bfdaa5e8d694df700395aabba380140b89ef2R211